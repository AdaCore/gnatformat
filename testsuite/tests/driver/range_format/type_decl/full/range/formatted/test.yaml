description: ConcreteTypeDecl formatting for line selection.
driver: gnatformat
args:
  [
    "test.ads",
    "--range-format",
    "-SL 39",
    "-EL 40",
    "--pipe",
  ]
