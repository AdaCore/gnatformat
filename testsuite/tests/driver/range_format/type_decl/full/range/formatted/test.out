test.ads(<ConcreteTypeDecl ["My_Int"] test.ads:39:4-39:36>) - 39:1-39:36
^
   type My_Int is range 0 .. 1_000;$

